docs/wiki/deployment/process-auditing.md

```diff
@@ -2,14 +2,75 @@
+propagated to the other.
+
+### Examine configuration flags
+
+```
+osquery> select * from osquery_flags where name in ("disable_events", "disable_audit");
++----------------+------+---------------------------------------------------+---------------+-------+------------+
+| name           | type | description                                       | default_value | value | shell_only |
++----------------+------+---------------------------------------------------+---------------+-------+------------+
+| disable_audit  | bool | Disable receiving events from the audit subsystem | true          | false | 0          |
+| disable_events | bool | Disable osquery publish/subscribe system          | false         | false | 0          |
++----------------+------+---------------------------------------------------+---------------+-------+------------+
+```
+
+### Examine event table
+
+osquery keeps state about the events subsystem in the `osquery_events`
+table. The `subscriptions` is of note here. This example is from a
+macOS machine with events enabled. (But no events detected)
+
+```
+osquery> select * from osquery_events;
++-------------------------+-----------------+------------+---------------+--------+-----------+--------+
+| name                    | publisher       | type       | subscriptions | events | refreshes | active |
++-------------------------+-----------------+------------+---------------+--------+-----------+--------+
+| diskarbitration         | diskarbitration | publisher  | 1             | 0      | 0         | 1      |
+| event_tapping           | event_tapping   | publisher  | 1             | 0      | 0         | 0      |
+| fsevents                | fsevents        | publisher  | 0             | 0      | 24        | 1      |
+| iokit                   | iokit           | publisher  | 1             | 0      | 0         | 1      |
+| openbsm                 | openbsm         | publisher  | 9             | 0      | 0         | 0      |
+| scnetwork               | scnetwork       | publisher  | 0             | 0      | 0         | 0      |
+| disk_events             | diskarbitration | subscriber | 1             | 0      | 0         | 1      |
+| file_events             | fsevents        | subscriber | 0             | 0      | 0         | 1      |
+| hardware_events         | iokit           | subscriber | 1             | 0      | 0         | 1      |
+| process_events          | openbsm         | subscriber | 8             | 0      | 0         | 1      |
+| user_events             | openbsm         | subscriber | 1             | 0      | 0         | 1      |
+| user_interaction_events | event_tapping   | subscriber | 1             | 0      | 0         | 1      |
+| yara_events             | fsevents        | subscriber | 0             | 0      | 0         | 1      |
++-------------------------+-----------------+------------+---------------+--------+-----------+--------+
+```
+
+### Try to get working on osqueryi
```

Comment:
nitpick, consider rephrasing

Reply:
Additionally, I think this section could be dropped or used as an introduction with general troubleshooting.

Reply:
agreed.